prombench/manifests/cluster-infra/grafana_dashboard_dashboards_noparse.yaml

```diff
@@ -1573,36 +1573,36 @@ data:
               "steppedLine": false,
               "targets": [
               {
-                  "expr": "node_memory_MemTotal_bytes{job=\"node-exporter\",namespace=\"prombench-[[pr-number]]\",node=~\"(test+).*\"}",
+                  "expr": "node_memory_MemTotal_bytes{job=\"node-exporter\",namespace=\"prombench-${prNumber}\",node=~\"(test+).*\"}",
```

Comment:
I don't understand why we're seeing this diff 🤨 

In master I see this line with `${prNumber}`

https://github.com/prometheus/test-infra/blob/475461d4fe2cd957af054c63e36b77d97eec4a67/prombench/manifests/cluster-infra/grafana_dashboard_dashboards_noparse.yaml#L1576

Same thing for all other changes in this file

Reply:
change was introduced here: https://github.com/prometheus/test-infra/commit/506588a5d256f983cb2060dd6cdd1e3b76456f75 maybe your local master was cloned before this one got in?

Think a local rebase to latest master should fix it?

Reply:
That was it, thanks :)